mysql-test/r/mysqltest.result

```diff
@@ -1103,7 +1103,7 @@ Host	localhost
 db	test
 Command	Query
 Time MASKED
-State	starting
```

Comment:
Which particular change dis cause this?

Reply:
This is because of upstream commit 4a7ae675095f7403510ba7f5b07efed6e2d4bbc5.
Execution path of SHOW PROCESSLIST changed and now  THD_STAGE_INFO(thd, stage_init); is called inside bool Sql_cmd_dml::execute(THD *thd)